static2/builtin/analyzer.py

```diff
@@ -15,9 +15,6 @@ def analyze_functions(static):
 # runs the recursive descent parser at address
 # how to deal with block groupings?
 def make_function_at(static, address, recurse = True):
-  if static['arch'] != "i386" and static['arch'] != "x86-64":
```

Comment:
QIRA supports many arches BAP does not. if ARM support works, add a check for ARM here as well.